src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -102,16 +109,48 @@ abstract contract DelegationManagerStorage is IDelegationManager {
      */
     mapping(IStrategy => uint256) public strategyWithdrawalDelayBlocks;
 
-    constructor(IStrategyManager _strategyManager, ISlasher _slasher, IEigenPodManager _eigenPodManager) {
+    /**
+     * @notice Global minimum withdrawal delay for all strategy withdrawals.
+     * NOTE: This is used equivalently to minWithdrawalDelayBlocks and strategyWithdrawalDelayBlocks but
+     * using timestamps instead in the Slashing release.
+     * In addition, we now also configure withdrawal delays on a per-strategy basis.
+     * To withdraw from a strategy, max(minWithdrawalDelay, strategyWithdrawalDelay[strategy]) number of seconds must have passed.
+     * See mapping strategyWithdrawalDelay below for per-strategy withdrawal delays.
+     */
+    uint256 public minWithdrawalDelay;
```

Comment:
let's make this immutable imo

Reply:
idk 

we should avoid needing to upgrade to change a config value